lib/Containers/Concurrent/thread.h

```diff
@@ -36,8 +36,7 @@ struct ThreadId {
 };
 template<typename Inspector>
 auto inspect(Inspector& f, ThreadId& x) {
-  return f.object(x).fields(f.field("LWPID", x.kernel_id),
-                            f.field("name", x.name()));
+  return f.object(x).fields(f.field("LWPID", x.kernel_id));
```

Comment:
Is there a situation where having the `posix_id` in addition to `kernel_id` might be helpful to know while we don't have the name?

Reply:
As far as I understood, the posix id does not provide more information, but it can act as a handle to do more stuff with it via the posix API. The lwpid is just a low level thread id of the kernel. So perhaps it can be useful to additionally have the posix id as well. I'll add it back.

Reply:
It's fine by me without it, it really was an open question. The cases I can think of I'd like to see the LWPID, and can't think of something where I'd need the posix_id.